File: cs/sample/Sample.Tests/Roman2Int/testRoman2Int.cs
using System;
using Xunit;
using Sample.Roman2Int;

namespace Sample.Tests{
    public class UnitTest_Roman2Int{
        [Theory]
        [InlineData("III",3)]
        [InlineData("IV",4)]
        [InlineData("IX",9)]
        [InlineData("LVIII",58)]
        [InlineData("MCMXCIV",1994)]
        public void checkResult(string s,int expected){
            Assert.Equal(expected,roman2Int.naive(s));
        }
    }
}

File: py/tests/testRoman2Int/test_Roman2Int.py
import unittest
from parameterized import parameterized as p
from solns.roman2Int.roman2Int import *

class TestRomain2Int(unittest.TestCase):
    @p.expand([
        ["III",3],["IV",4],["IX",9],["LVIII",58],["MCMXCIV",1994]
    ])
    def test_naive(self,s,expected):
        self.assertEqual(Solution.naive(s),expected)